cli/src/main/java/com/devonfw/tools/ide/tool/npm/Npm.java

```diff
@@ -22,9 +26,41 @@ public Npm(IdeContext context) {
     super(context, "npm", Set.of(Tag.JAVA_SCRIPT, Tag.BUILD));
   }
 
+  @Override
+  protected VersionIdentifier computeInstalledVersion() {
+    if (hasNodeBinary("npm")) {
+      return VersionIdentifier.of(this.context.newProcess().runAndGetSingleOutput("npm", "--version"));
+    }
+    return null;
+  }
+
+  @Override
+  protected void postInstall() {
+    super.postInstall();
+    ProcessResult result = runPackageManager("config", "set", "prefix", getToolPath().toString());
+    if (result.isSuccessful()) {
+      this.context.success("Setting npm config prefix to: {} was successful", getToolPath());
+    }
+    VersionIdentifier version = this.context.getNpmRepository().resolveVersion(this.tool, getConfiguredEdition(), getConfiguredVersion(), this);
+    runPackageManager("install", "-g", "npm@" + version);
```

Comment:
The method `performToolInstallation` is already doing this.
Why would we install the tool itself again in `postInstall()`?
```suggestion
```